dominion/dominion_ui/src/Theme.js

```diff
@@ -38,6 +38,9 @@ const Theme = createMuiTheme({
       fontFamily: FONT_FAMILY_HEADING,
       textTransform: 'capitalize'
     },
+    body1: {
+      color: '#fff'
```

Comment:
This is best set in the component itself since by default, `color` should be black-ish to fit everything else outside of the this header.